### DISCUSSION TITLE
I've open 3 account for test differents configuration ...

### DISCUSSION THREAD

**Discussion by pierrealexaline** [revoxhere/duino-coin#1351] [Q&A]
Hello there, i've open 3 differents account for test differents configurations, with arduino cards, esp32, esp1866,Arduino Mega,RPI 3 and 4, etc ... I would like to test the exchanges between the wallets accounts, but i just read that :
"Having alt accounts without a rational reason (e.g. for multi-mining) are not allowed."
My first question is that considered as a rational reason ? 
my second question : If it's not, how can i delete the alt accounts ? and give duco to faucet for exemple ?
thanks for the response ! and great work !

**Comment by pierrealexaline** [comment]
"An error has occurred while sending funds: You're using multiple accounts..."....what can i do now with that message ? thx for answer !

**Comment by Bilaboz** [comment]
Hello sorry for the late answer, what is your main account that you want to keep please?

**Reply by pierrealexaline** [reply]
Hello ! EuskalMiner is the name of the account i want to keep if it's possible. 
LozerianTeam and BaionaFull can be deleted ! thx by advance !
Nota : is it possible to have multi account only for developing applications like game, ...etc
Suggestion : it will be nice to have the possibility to change profile informations and the possibility to delete an account (i think about RGPD in EU).

**Reply by JulioniMigoyeas** [reply]
Hello, I have the exact same problem of the multiple accounts and now I can't do anything with my coins. Could you erase ir for me, please? The account I want to keep is Julioni, and the one I want to remove is Migo.
Thanks!!!

**Comment by Sule270774** [comment]
it's useless and pointless to post questions here with no answers or solutions..! So that's what this forum was created for
